src/styles/data-list.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import EditOutlinedIcon from '@material-ui/icons/EditOutlined';
+import DeleteOutlineOutlinedIcon from '@material-ui/icons/DeleteOutlineOutlined';
+
+export const Container = styled.div`
```

Comment:
could you pelase enrich the name so that we won't confuse it with the material UI Container component?